wdl2cwl/tests/cwl_files/BuildCembaReferences.cwl

```diff
@@ -0,0 +1,397 @@
+        source: IndexReverse.index_prefix
+      - id: monitoring_script
+        source: IndexReverse.monitoring_script
+    out:
+      - id: bowtie2_index_files
+      - id: monitoring_log
+    run:
+        class: CommandLineTool
+        id: Bowtie2Build
+        inputs:
+          - id: fasta_input
+            type: File
+          - id: index_prefix
+            type: string
+          - id: monitoring_script
+            type:
+              - File
+              - 'null'
+        outputs:
+          - id: bowtie2_index_files
+            type:
+                items: File
+                type: array
+            outputBinding:
+                glob: $(inputs.index_prefix + "*")
+          - id: monitoring_log
+            type: File
+            outputBinding:
+                glob: monitoring.log
+        requirements:
+          - class: DockerRequirement
+            dockerPull: quay.io/broadinstitute/bowtie2:2.3.4.3
+          - class: InitialWorkDirRequirement
+            listing:
+              - entryname: example.sh
+                entry: |4
+
+                    # if the WDL/task contains a monitoring script as input
+                    if [ ! -z "$(inputs.monitoring_script === null ? "" : inputs.monitoring_script.path)" ]; then
+                      chmod a+x $(inputs.monitoring_script === null ? "" : inputs.monitoring_script.path)
+                      $(inputs.monitoring_script === null ? "" : inputs.monitoring_script.path) > monitoring.log &
+                    else
+                      echo "No monitoring script given as input" > monitoring.log &
+                    fi
+
+                    bowtie2-build \
+                      -f $(inputs.fasta_input.path) $(inputs.index_prefix)
+          - class: InlineJavascriptRequirement
+          - class: NetworkAccess
+            networkAccess: true
+          - class: ResourceRequirement
+            coresMin: 1
+            ramMin: 6675.72021484375
+            outdirMin: 1024
+        cwlVersion: v1.2
+        baseCommand:
+          - bash
+          - example.sh
+  - id: CreateReferenceDictionary
+    in:
+      - id: reference_fasta
+        source: CreateReferenceDictionary.reference_fasta
+      - id: monitoring_script
+        source: CreateReferenceDictionary.monitoring_script
+    out:
+      - id: ref_dict_output
+      - id: monitoring_log
+    run:
+        class: CommandLineTool
+        id: CreateReferenceDictionary
+        inputs:
+          - id: reference_fasta
+            type: File
+          - id: monitoring_script
+            type:
+              - File
+              - 'null'
+        outputs:
+          - id: ref_dict_output
+            type: File
+            outputBinding:
+                glob: "$(inputs.ref_dict_output_name === null ? (inputs.reference_fasta.basename.replace(/\\\
+                    .fa$/, '') ).split('/').reverse()[0].replace(/\\.fasta$/, '')\
+                    \ + \".dict\" : inputs.ref_dict_output_name)"
```

Comment:
There is no `inputs.ref_dict_output_name`; there is a WDL declaration called `ref_dict_output_name`, yes, but that is not in the `inputs` section of this WDL Task: https://github.com/broadinstitute/warp/blob/b3f5c898d1b535aaca8dc9e9f4223697a7a0a9a9/pipelines/cemba/build_cemba_references/BuildCembaReferences.wdl#L149

```suggestion
                glob: "($inputs.reference_fasta.basename.replace(/\\.fa$/, '') ).split('/').reverse()[0].replace(/\\.fasta$/, '') + ".dict")
```
This is a conversion of a WDL declaration, not a WDL expression placeholder. The quotation marks should not be quoted